docs/install.rst

```diff
@@ -113,7 +113,7 @@ version of the Photutils source code can be retrieved using git::
 Then to build and install Photutils, run::
 
     cd photutils
-    python setup.py install
+    pip install .[all]
```

Comment:
maybe have `pip install -e .[all]` ?  I just assume that most people who want a dev version want it to be able to tinker with it?

Reply:
Sounds good, thanks!  I'll mention that as an option for editable installs.

Reply:
done!